api/include/opentelemetry/propagators/composite_http_propagator.h

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020, OpenTelemetry Authors
```

Comment:
It doesn't look like CompositeHTTPPropagator is used anywhere in this PR.  Can you take it out for now to make the PR smaller?  I expect less to review means a quicker review process.